doc/api/addons.md

```diff
@@ -107,6 +107,9 @@ When building addons with `node-gyp`, using the macro `NODE_GYP_MODULE_NAME` as
 the first parameter of `NODE_MODULE()` will ensure that the name of the final
 binary will be passed to `NODE_MODULE()`.
 
+If you define addon with `NODE_MODULE`, the addon can not be loaded both in the
```

Comment:
Is it working either in main or in one worker or only in the main thread?

Reply:
My understanding is it works in main or in one worker. I think the point is if the addon is defined by `NODE_MODULE`, it only can be loaded once.

Reply:
But if it is loaded in worker and then the worker exits, it can be loaded again.

Reply:
I have updated the docs.